apps/actions-dependencies-updater/src/caches.mts

```diff
@@ -0,0 +1,95 @@
+import * as log from "./logger.mjs";
+import { GithubShaToVersionCache } from "./github.mjs";
+import { UpdateTransaction } from "./updater.mjs";
+
+const CACHE_DIR = ".cache/";
```

Comment:
So this will create a `.cache` in the package directory, since we're depending on `pnpm` for execution?

Reply:
It's nice to cache this if you're running it for multiple repos in succession. The longest part of the execution is fetching from github.

I don't know what you mean by "since we're depending on pnpm for execution".

Reply:
Using pnpm for executing the script will always set the cwd to where the package manifest lives. I was asking just to confirm that, since usually we'll package up something like this with https://github.com/vercel/ncc

Reply:
We can also do that - what do you think is the best way?

Reply:
I wouldn't bother with binary packaging if there's an urge to get this running. It can cause its own problems when it comes to building for different platforms and binary distributions. If this becomes high demand we should probably merge this into `clunk` instead which already has a CLI framework and binary distribution setup